src/patternfly/components/Page/page.scss

```diff
@@ -65,6 +65,16 @@
   --pf-c-page__header-tools--c-avatar--MarginLeft: var(--pf-global--spacer--md);
   --pf-c-page__header-tools-group--MarginLeft: var(--pf-global--spacer--xl);
 
+  // Header tools group button
+  --pf-c-page__header-tools-group--m-icons--c-button--Width: #{pf-size-prem(36px)};
+  --pf-c-page__header-tools-group--m-icons--c-button--Height: #{pf-size-prem(36px)};
+  --pf-c-page__header-tools-group--m-icons--c-button--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor: transparent;
+  --pf-c-page__header-tools-group--m-icons--c-button--active--BackgroundColor: var(--pf-global--BackgroundColor--dark-200);
+  --pf-c-page__header-tools-group--m-icons--c-button--focus--BackgroundColor: var(--pf-global--BackgroundColor--dark-200);
+  --pf-c-page__header-tools-group--m-icons--c-button--focus__c-notification-badge--m-unread--after--BorderColor: var(--pf-global--BackgroundColor--dark-200);
+  --pf-c-page__header-tools-group--m-icons--c-button--active__c-notification-badge--m-unread--after--BorderColor: var(--pf-global--BackgroundColor--dark-200);
+
```

Comment:
Looks like we need `--pf-c-page__header-tools-group--m-icons--c-button--PaddingLeft` and `--pf-c-page__header-tools-group--m-icons--c-button--PaddingRight`.

Reply:
Ah I had unset padding and given the button a width and height instead... just changed it so that it uses `:after` to draw the circle background

Reply:
The problem with setting a strict `height/width` is that if `font-size` changes the icon size it breaks the background treatment.